Cargo.toml

```diff
@@ -28,8 +28,8 @@ dot-out = []
 glsl-in = ["pp-rs"]
 glsl-out = []
 msl-out = []
-serialize = ["serde", "bitflags/serde", "indexmap/serde-1"]
-deserialize = ["serde", "bitflags/serde", "indexmap/serde-1"]
+serialize = ["serde", "bitflags/serde", "indexmap/serde"]
+deserialize = ["serde", "bitflags/serde", "indexmap/serde"]
```

Comment:
This code LGTM for most Rust consumers. However, I'm not sure if Firefox can tolerate these bumps in `serde` (1 → 2) and `indexmap` (1 → 2) yet. @jimblandy, can you help me understand how I could answer this question?